lib/manageiq_performance/middlewares/active_record_queries.rb

```diff
@@ -130,12 +132,19 @@ def should_measure?
           Thread.current[:miq_perf_sql_query_data]
         end
 
+        def include_queries?
+          return @include_queries if defined?(@include_queries)
+          @include_queries = ManageIQPerformance.config.include_sql_queries?
```

Comment:
What advantage are we gaining by moving this two a 2 line statement?  You do this on the following two methods as well.